.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test sros2
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
```

Comment:
Also, I have found this to be of limited usefulness, as we discussed. No strong opinions though.

Reply:
on some leaf projects this actually helped us keep up with API and behavior changes.

Arguably this is part of `ros2.repos` so I guess there are less surprises of API changes needing to be cought by this CI.

I'll keep it so that we (not sure who "we" is in this context) receive emails automatically if it starts failing

Reply:
Very good, let's leave it then.